AppinfoJsonFile.js

```diff
@@ -114,11 +114,12 @@ AppinfoJsonFile.prototype.makeKey = function(source) {
 };
 
 AppinfoJsonFile.prototype.loadSchema = function(source) {
-    var localizeProperties = {}, schemaFilePath;
+    var localizeProperties = {}, schemaFilePath, modulePath;
     if (this.project.schema) {
         schemaFilePath = path.join(process.env.PWD, this.project.schema);
     } else {
-        schemaFilePath = path.join(process.env.PWD, "node_modules", "ilib-loctool-webos-appinfo-json", "schema/appinfo.schema.json");
+        modulePath = path.join(require.resolve("ilib-loctool-webos-appinfo-json"), "../");
```

Comment:
`modulePath = path.parent(__dirname);`

See https://nodejs.org/api/modules.html#modules_dirname

Reply:
I've updated to code to used `__dirname`. If I used it, I don't need to get a parent path. Thank you!